### PR TITLE
Removed options.response after usage to prevent uncessary short circuit

### PR DESCRIPTION
**Pull request by strworkstation**
#123

### CODE REVIEW

File: test/urlhandler.coffee [hunk 1]
```diff
@@ -8,10 +8,13 @@ urlfor = (relpath) ->
 describe 'URLHandler', ->
     describe '#get', ->
         it 'should return options.response when it\'s provided', (done) =>
-            URLHandler.get urlfor('sample.xml'), {response: 'response'}, (err, xml) ->
+            options = {response: 'response'}
+            URLHandler.get urlfor('sample.xml'), options, (err, xml) ->
                 should.not.exist err
+                console.log xml
```
Comment (michaelvial) [inline test/urlhandler.coffee]: Can you get rid of the `console.log`

File: src/urlhandler.coffee [hunk 2]
```diff
@@ -10,7 +10,9 @@ class URLHandler
 
         if options.response?
             # Trick: the VAST response XML document is passed as an option
-            cb(null, options.response)
+            tempResponse = options.response
+            options.response = undefined
```
Comment (michaelvial) [inline src/urlhandler.coffee]: Can you change it to:
``` coffeescript
response = options.response
delete options.response
cb(null, response)
```

**Comment by rgulrajani** [comment]
Is this going to be part of the 1.3.10 branch?

**Comment by michaelvial** [comment]
@rgulrajani yes, we're planing to release a new version tomorrow, probably a 1.4.0 tho.

**Comment by rgulrajani** [comment]
Thanks for the quick reply @michaelvial

**Comment by strworkstation** [comment]
@michaelvial changes made!

**Comment by michaelvial** [comment]
Thx @strworkstation. I've forgotten to say, but can you merge your commit into a single one? Then I'll merge it.

**Pull request merged**